### PR TITLE
Fix pagination

### PR DESCRIPTION
**Pull request by dougdroper**
Moving the grouping into an SQL query and pushing the pagination down

### CODE REVIEW

File: app/presenters/queries/content_item_presenter.rb [hunk 1]
```diff
@@ -24,20 +26,43 @@ def initialize(content_item_scope, fields = nil)
...
+        ActiveRecord::Base.connection.execute(aggregated_sql(scope.to_sql))
+      end
+
+      def aggregated_sql(sql)
+        <<-END.strip_heredoc
+          SELECT json_agg(json_rows) FROM (
+            SELECT row_to_json(item) json_item FROM (#{sql}) item
+          ) json_rows
+          GROUP BY json_item->>'content_id', json_item->>'locale'
+          ORDER BY max(json_item->>'public_updated_at') DESC
```
Comment (danielroseman) [inline app/presenters/queries/content_item_presenter.rb]: Since `order` is a parameter we should interpolate it here rather than hard-coding.

**Comment by danielroseman** [comment]
This is also affecting the linkables endpoint - although that does not pass in the page or per_page params, it is still being limited to 50 items only. That endpoint always needs to return all items.

**Comment by dougdroper** [comment]
@danielroseman, sure, will sort that out. :+1:

**Comment by dougdroper** [comment]
@danielroseman the linkables endpoint is now no longer limited.

**Pull request merged**